stock_prices/stock_prices.py

```diff
@@ -3,7 +3,23 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  smallest = 0
+  smallestIdx = 0
+  biggest = 0
+  biggestIdx = 0
+
+  for index, value in enumerate(prices):
```

Comment:
i like that you're using other Python features! I'd never seen enumerate before
however in this case, I think it may be easier if you simply use a for loop and get the values by checking `prices[index]`

Reply:
great job keeping this at O(n)!